woocommerce-services.php

```diff
@@ -747,44 +747,6 @@ protected function add_method_to_shipping_zone( $zone_id, $method_id ) {
 			$this->nux->dismiss_pointer( 'wc_services_add_service_to_zone' );
 		}
 
-		public function init_core_wizard_shipping_config() {
-			if ( $this->is_wc_shipping_activated() ) {
-				return;
-			}
```

Comment:
This methods looks for options set by the WC setup wizard to preconfigure shipping methods. The options it looks for were [removed in WC 3.6](https://github.com/woocommerce/woocommerce/pull/21645) and are no longer set by newer versions of WC. This plugin requires WC ≥ 8.8.

Instead of conditionally loading this, I removed it altogether.